gradle/plugins/src/main/kotlin/com.hedera.gradle.spotless.gradle.kts

```diff
@@ -25,7 +25,16 @@ spotless {
 
     format("misc") {
```

Comment:
With the new pattern – `**/*.md` – and this being applied to the root project **and** each subproject, the same files will be formatted several times in parallel. This leads to Gradle errors. 

As we have files that are not associated with any Subproject in the docs folders, I would suggest to just have one "spotlessMisc" task in the _root_. We do the same for formatting `.gradle.kts` files that are not associated with a project.

For this, I would suggest:
- Move the `format("misc")` to a new file and maybe als name it `format("md")` for clarity -> 
`com.hedera.gradle.spotless-md.gradle.kts`
- Then use that **only** in the root project
add `id("com.hedera.gradle.spotless-md")`to `com.hedera.gradle.root.gradle.kts`

Reply:
Have implemented. Thanks for the suggested fix @jjohannes ✅